src/hyperion/parameters/rotation.py

```diff
@@ -104,3 +116,52 @@ def scan_points(self) -> AxesPoints:
     @property
     def num_images(self) -> int:
         return int(self.scan_width_deg / self.rotation_increment_deg)
+
+
+class MultiRotationScan(RotationScanGeneric, SplitScan):
+    rotation_scans: Annotated[list[RotationScanCore], Len(min_length=1)]
+
+    def _single_rotation_scan(self, scan: RotationScanCore) -> RotationScan:
+        params = self.dict()
+        del params["rotation_scans"]
+        params.update(scan.dict())
+        return RotationScan(**params)
+
+    @root_validator(pre=False)
+    def validate_snapshot_directory(cls, values):
+        start_img = 0
+        for scan in values["rotation_scans"]:
+            scan.nexus_vds_start_img = start_img
+            start_img += scan.scan_width_deg / values["rotation_increment_deg"]
+        return values
+
+    @property
+    def single_rotation_scans(self) -> Iterator[RotationScan]:
+        for scan in self.rotation_scans:
+            yield self._single_rotation_scan(scan)
+
+    def _num_images_per_scan(self):
+        return [
+            int(scan.scan_width_deg / self.rotation_increment_deg)
+            for scan in self.rotation_scans
+        ]
+
+    @property
+    def num_images(self):
+        return sum(self._num_images_per_scan())
+
+    @property
+    def scan_points(self):
```

Comment:
Is there a GH issue we can reference here?

Reply:
https://github.com/DiamondLightSource/hyperion/issues/1491 (added to the comment too)